js/MainPage/CascadeMain.js

```diff
@@ -313,8 +331,8 @@ function initialize(projectContent = null) {
                     gui = new ControlKit({ parentDomElementId: "controlKit" });
                 } else {
                     // We are loading a new project, controlKit needs to have
-                    // it's node ovirriden with the new element
-                    gui._node._element = $('#controlKit')[0];
```

Comment:
I think this was the only line that used jquery, I didn't think this line warranted having the extra dependency to converted to vanilla js.

Reply:
Ah yeah, that one must have snuck in with brad's PR; jQuery was mainly only used in GoldenLayout.  No problem 👍